foreman-selinux-enable

```diff
@@ -13,15 +13,22 @@ trap "rm -rf '$TMP'" EXIT INT TERM
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    # UPGRADE STEP for Foreman 1.8 (remove after release)
+    # If katello policy was not yet loaded and elasticsearch port is still defined,
+    # remove it (it was moved to katello policy). Katello policy will not load before
+    # we remove the port definition (because SELinux prevents that).
+    semodule -l | grep -q katello || \
```

Comment:
Requiring it is a problem though.  As the current released katello-selinux requires it without defining it, then this will happen I think:

1. foreman-selinux-enable runs
    1. does not remove elasticsearch_port_t as 'katello' module is loaded
    2. loads new 'foreman' module which no longer defines it, this will throw an error

While what should have happened, if the current released katello-selinux wasn't using it or didn't exist (the scenario you tested) is:

1. foreman-selinux-enable runs
    1. removes elasticsearch port range as 'katello' isn't loaded
    2. loads new 'foreman' module which no longer defines it
2. katello-selinux-enable runs
    1. loads new 'katello' module which defines it
    2. adds elasticsearch_port_t range